lib/boat_tracker.rb

```diff
@@ -0,0 +1,34 @@
+class BoatTracker
+
+  def self.any_boat_hit?(boat_list, target_coords)
```

Comment:
You're passing boat_list to all methods - maybe this is the state of a BoatTracker instance?